packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm.tsx

```diff
@@ -145,17 +152,86 @@ export default function DataAccessRequestAccessorsFilesForm(
     })
   }
 
-  const { mutate: submit } = useSubmitDataAccessRequest({
-    onSuccess: () => {
-      onSubmissionCreated()
-    },
-    onError: onError,
-  })
+  const { mutate: submit, isLoading: isLoadingSubmitDataAccessRequest } =
+    useSubmitDataAccessRequest({
+      onSuccess: () => {
+        onSubmissionCreated()
+      },
+      onError: onError,
+    })
 
-  const { mutateAsync: updateRequestAsync } = useUpdateDataAccessRequest({
+  const {
+    mutateAsync: updateRequestAsync,
+    isLoading: isLoadingUpdateDataAccessRequest,
+  } = useUpdateDataAccessRequest({
     onError: onError,
   })
 
+  const isLoading =
+    isLoadingGetDataAccessRequest ||
+    isLoadingUpdateDataAccessRequest ||
+    isLoadingSubmitDataAccessRequest
+
```

Comment:
Track loading state so we can prevent updates while syncing changes to the server